h2/src/main/org/h2/mvstore/DataUtils.java

```diff
@@ -941,8 +941,8 @@ public static int getErrorCode(String m) {
      * @return the parsed value
      * @throws IllegalStateException if parsing fails
      */
-    public static long readHexLong(Map<String, ? extends Object> map,
-            String key, long defaultValue) {
+    public static long readHexLong(Map<String, ?> map,
+                                   String key, long defaultValue) {
```

Comment:
Do not use such indentation, it will break build process of H2. Just add 8 more spaces for wrappend lines, and 4 spaces in other cases. Adjust code formatting settings in your IDE for H2 if it has such options.

Line length limit for code is 120 characters now (80 for comments), so line break here is not required (and not forbidden).